Documentation/packaging-installation.md

```diff
@@ -0,0 +1,82 @@
+# .NET Core Packaging and Installation
+
+This document helps you install or package a .NET Core SDK built using source-build.
+
+The SDK built by source-build is generally not portable. That means it will work on the same operating system where it was built. It will not work on older operating systems. It may work on newer operating systems.
+
+The built SDK is generally located at `artifacts/${ARCHITECTURE}/Release/dotnet-sdk-${SDK_VERSION}-${RUNTIME_ID}.tar.gz`.
+
+## Using the SDK directly (install per-user)
+
+To use the SDK directly, unpack the SDK to any directory and then use the `dotnet` executable from it.
+
+You can find more details on how to manually install an SDK from a tarball at [manually installing an SDK](https://docs.microsoft.com/en-us/dotnet/core/install/linux-ubuntu#manual-install). The same steps listed should work for all Linux distributions where `bash` is the default shell.
+
+## Installing the SDK globally
+
+If you want to install the SDK globally (and not per user), here are some suggestions.
+
+1. Extract the tarball to a distribution-appropriate location such as `/usr/local/lib64/dotnet/`, `/usr/local/lib/dotnet/`, or `/usr/local/lib/x86_64-linux-gnu/dotnet/`.
+
+2. Create a symlink from `/usr/local/bin/dotnet` (or equivalent location available in `$PATH`) to the `dotnet` binary in SDK that you installed in the previous step. Foe example, `/usr/local/bin/dotnet` -> `/usr/local/lib64/dotnet/dotnet`. This lets users simply run `dotnet` and have it work.
+
+3. Create an `/etc/dotnet/install_location` file and add the path of the SDK directory in there (such as `/usr/local/lib64/dotnet`). This file is used by [.NET Core to find the SDK/Runtime location](https://github.com/dotnet/designs/blob/master/accepted/2020/install-locations.md).
```

Comment:
A bit more specific what should be in the file? (And "add" is a little confusing IMO since it's a new file, I usually look for an example/pattern to follow when I see "add".)

```suggestion
3. Create an `/etc/dotnet/install_location` file with a single line of content, the path of the SDK directory (such as `/usr/local/lib64/dotnet`). This file is used by [.NET Core to find the SDK/Runtime location](https://github.com/dotnet/designs/blob/master/accepted/2020/install-locations.md).
```